### PR TITLE
vhacd: Properly identify downstream changes, fix MinGW build issue

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: thirdparty/README.md [hunk 1]
```diff
@@ -297,6 +298,7 @@ File extracted from upstream release tarball `mbedtls-2.16.0-apache.tgz`:
 - Applied the patch in `thirdparty/mbedtls/1453.diff` (PR 1453). Soon to be merged upstream. Check it out at next update.
 - Applied the patch in `thirdparty/mbedtls/padlock.diff`. This disables VIA padlock support which defines a symbol `unsupported` which clashses with a symbol in libwebsockets.
```
Comment (fire) [inline thirdparty/README.md]: clashses -> clashes.

Comment (akien-mga) [reply thirdparty/README.md]: That's not from this commit, but sure, I can fix that too.

Comment (akien-mga) [reply thirdparty/README.md]: Fixed.

File: thirdparty/README.md [hunk 2]
```diff
@@ -294,8 +295,12 @@ Godot build configurations, check them out when updating.
 File extracted from upstream release tarball `mbedtls-2.16.0-apache.tgz`:
 - All `*.h` from `include/mbedtls/` to `thirdparty/mbedtls/include/mbedtls/`
 - All `*.c` from `library/` to `thirdparty/mbedtls/library/`
-- Applied the patch in `thirdparty/mbedtls/1453.diff` (PR 1453). Soon to be merged upstream. Check it out at next update.
-- Applied the patch in `thirdparty/mbedtls/padlock.diff`. This disables VIA padlock support which defines a symbol `unsupported` which clashses with a symbol in libwebsockets.
+- Applied the patch in `thirdparty/mbedtls/1453.diff` (PR 1453).
+  Soon to be merged upstream. Check it out at next update.
```
Comment (fire) [inline thirdparty/README.md]: Grammar problems in this area. 

Check it out at next update.
->
Check it out at the next update.

:(

Comment (akien-mga) [reply thirdparty/README.md]: That's not really important, this will be removed at the next update :)

**Pull request merged**